generic/ras.py

```diff
@@ -40,6 +40,14 @@ def setUp(self):
         architecture = os.uname()[4]
         if "ppc" not in architecture:
             self.skip("supported only on Power platform")
+        if 'PowerNV' in process.system_output('grep ^platform /proc/cpuinfo'):
```

Comment:
Could you please use the python's builtin `open()`? It's way more efficient for that operation:
```
$ time python -c "import avocado; print 'GenuineIntel' in avocado.utils.process.system_output('grep GenuineIntel /proc/cpuinfo')"
True

real    0m0.178s
user    0m0.155s
sys     0m0.023s
```
```
$ time python -c "print 'GenuineIntel' in open('/proc/cpuinfo', 'r').read()"
True

real    0m0.033s
user    0m0.027s
sys     0m0.006s
```